src/test/java/org/alfresco/rest/api/tests/AuthenticationsTest.java

```diff
@@ -185,13 +196,44 @@ public void testCreateValidateDeleteTicket() throws Exception
 
         // Get list of site by appending the invalidated ticket
         getAll(SiteEntityResource.class, paging, ticket, 401);
-
-
+    }
+        /**
```

Comment:
I suspect there is indentation problem somewhere, see the full file:
https://github.com/Alfresco/alfresco-remote-api/blob/10a5bbb1f34c7e0484ad4dfe1db100388fbb8628/src/test/java/org/alfresco/rest/api/tests/AuthenticationsTest.java